### PR TITLE
Supports spaces between words.

### PR DESCRIPTION
**Pull request by yoshi389111**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,8 +122,8 @@ <h1>\(\KaTeX \sf ルビジェネレーター \space for \space Misskey \space v1
         };
         $$('input,select').forEach(element => {
             element.addEventListener('input', () => {
-                const ruby = $$('#ruby').value;
-                const base = $$('#base').value;
+                const ruby = $$('#ruby').value.trim().replace(" ", "\\ ");
```
Comment (hidao80) [inline index.html]: I think this is very good!👍
I would also like to ask, could you change `replace()` to `replaceAll()`? 
I would like to support the second and subsequent spaces when there are more than 3 words.

とても良いと思います！
あと、お願いなのですが、`replace()`を`replaceAll()`に変えていただけないでしょうか？
3語以上ある場合の2つ目以降のスペースに対応したいです。

**Review by hidao80 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (yoshi389111) [reply index.html]: Thank you for pointing out.

Fixed `replase()` to `replaceAll()`.

**Review by hidao80 — APPROVED** [review]
LGTM👍

File: index.html (hunk 1 above)
Comment (hidao80) [reply index.html]: LGTM👍

**Pull request merged**